### PR TITLE
Add in MOTD support

### PR DESCRIPTION
**Pull request by norrismiv**

### CODE REVIEW

File: hybrasyl/ControlService.cs [hunk 1]
```diff
@@ -42,11 +45,24 @@ public string Shutdown(string key)
...
         public User User(string name)
         {
             return World.ActiveUsers.All(x => x.Value.Name != name) ? null : World.ActiveUsers.Single(x => x.Value.Name == name).Value;
         }
+
+        public static string GetMotd()
+        {
+            using (HttpClient client = new HttpClient())
+            {
+                client.BaseAddress = new Uri("http://build.hybrasyl.com/api/news");
```
Comment (baughj) [inline hybrasyl/ControlService.cs]: this should be an XML config variable

**Comment by norrismiv** [comment]
I was in the process of investigating how the wcf endpoint handles the interactions with the config xml vs app.config.

**Comment by baughj** [comment]
Whoops. I thought we were going to make those updates later. Reverted.

**Comment by baughj** [comment]
...I don't think it is possible to re-open the PR, unfortunately, but you can just open it again?

**Pull request merged**